tests/json/crud/arguments.rs

```diff
@@ -56,9 +56,7 @@ impl Arguments {
     pub fn aggregate_from_json(object: &Map<String, Value>) -> Result<Arguments, String> {
         let options = AggregateOptions::from_json(object);
 
-        let f = |x| Some(Bson::from_json(x));
-
-        let array = val_or_err!(object.get("pipeline").and_then(f),
+        let array = val_or_err!(object.get("pipeline").map(Bson::from_json),
```

Comment:
Ohhh, was this not a clippy thing? I would have been fairly impressed if clippy was able to deal with a closure and a method chain inside  macro in separate statements.